charts/library/common/templates/lib/dependencies/_mariadbInit.tpl

```diff
@@ -0,0 +1,23 @@
+{{/*
+This template ensures pods with postgresql dependency have a delayed start
+*/}}
+{{- define "common.dependencies.mariadb.init" -}}
+{{- $pghost := printf "%v-%v" .Release.Name "mariadb" }}
+{{- if .Values.mariadb.enabled }}
+- name: mariadb-init
+  image: "{{ .Values.mariadblImage.repository}}:{{ .Values.mariadblImage.tag }}"
+  securityContext:
+    capabilities:
+      drop:
+        - ALL
+  resources:
+  {{- with .Values.resources }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  command:
+    - "bash"
+    - "-ec"
+    - "until /opt/bitnami/scripts/mariadb/healthcheck.sh; do sleep 2; done"
```

Comment:
Are you sure this script actually targets the specific MariaDB instance you want to target with this?

Reply:
According to their comments on the script, it targets current instance
```
########################
# Execute a reliable health check against the current mysql instance
# Globals:
#   DB_ROOT_PASSWORD, DB_MASTER_ROOT_PASSWORD
# Arguments:
#   None
# Returns:
#   mysqladmin output
#########################
mysql_healthcheck() {
    local args=("-uroot" "-h0.0.0.0")
    local root_password


    root_password="$(get_master_env_var_value ROOT_PASSWORD)"
    if [[ -n "$root_password" ]]; then
        args+=("-p${root_password}")
    fi


    mysqladmin "${args[@]}" ping && mysqladmin "${args[@]}" status
}


```

Reply:
I see what you mean now. Will have to define the -h to target the correct container